onnx/reference/c_ops/__init__.py

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Should we add text to caution that we should avoid reference implementation in C unless it’s absolutely need like in this case?